src/main/java/com/softserveinc/ita/pageobjects/admin/MainMenu.java

```diff
@@ -4,33 +4,90 @@
 
 import static com.codeborne.selenide.Condition.appear;
 import static com.codeborne.selenide.Selenide.$x;
-import static java.lang.String.format;
+import static com.softserveinc.ita.pageobjects.admin.MainMenuButtons.*;
 import static java.time.Duration.ofSeconds;
 
 public class MainMenu {
 
-    private final String MENU_ITEM_PATH_TEMPLATE = "//a[@href='/admin/%s']";
+    @Step("Main menu: Opened dashboard page")
+    public DashboardPage openDashboardPage() {
+        DASHBOARD_PAGE
```

Comment:
create a private void method for opening the page

```
private void openPage(String pageName) {
        $x(format("//a[@href='/admin/%s']", pageName))
                .should(appear, ofSeconds(5))
                .click();
    }
```

and call it in such a way


```
@Step("Main menu: Opened dashboard page")
    public DashboardPage openDashboardPage() {
        openPage(DASHBOARD_PAGE.getPageName());

        return new DashboardPage();
    }
```

Reply:
thanks)